### PR TITLE
Black actions

### PR DESCRIPTION
**Pull request by cazapatamar**
## 🛠️ Changes
- Add black workflow for actions
- Delete flake config and actions

## 📝 Associated issues
#2

### CODE REVIEW

File: .github/workflows/black.yml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+name: Check code format
+
+on: [pull_request]
```
Comment (erikasv) [inline .github/workflows/black.yml]: Why not only in develop PR?

File: .github/workflows/black.yml [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+
+jobs:
+  lint:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out source repository
+        uses: actions/checkout@v3
+      - name: Black format check
```
Comment (erikasv) [inline .github/workflows/black.yml:18]: you're missing the step to configure the python version

**Review by erikasv — CHANGES_REQUESTED** [review]

File: .github/workflows/black.yml [hunk 3]
```diff
@@ -0,0 +1,15 @@
+name: Check code format
+
+on: [pull_request]
+
+jobs:
+  lint:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out source repository
+        uses: actions/checkout@v3
```
Comment (erikasv) [inline .github/workflows/black.yml]: update this version, see the stac-data-tools PR discussions

**Review by erikasv — CHANGES_REQUESTED** [review]

File: .github/workflows/black.yml (hunk 1 above)
Comment (cazapatamar) [reply .github/workflows/black.yml]: Done!

File: .github/workflows/black.yml (hunk 2 above)
Comment (cazapatamar) [reply .github/workflows/black.yml:18]: Done!

File: .github/workflows/black.yml (hunk 3 above)
Comment (cazapatamar) [reply .github/workflows/black.yml]: Fixed!

**Comment by erikasv** [comment]
Remove the other unused branches

**Comment by cazapatamar** [comment]
> Remove the other unused branches

Done!

**Review by erikasv — APPROVED** [review]

**Pull request merged**